mysite/account/templates/header.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
This template includes not only header but also others, the name shouldn't be `header.html`, the name likes `base.html`, ...